pennylane/ops/qubit.py

```diff
@@ -51,14 +52,21 @@
     QubitStateVector
 
 
+Observables
+-----------
+
+.. autosummary::
+    Hermitian
```

Comment:
was this not there before?

Reply:
What about pauli matrices and hadamard?

Reply:
They are above in the `Operations` section 😕 
Couldn't get Sphinx to list it twice without breaking/confusing Sphinx (it would also break links to the Pauli matrices)

Reply:
Oh, this is just the autosummary. It should work listing it twice here, I'll do that.

Reply:
Fixed